web_app/main.py

```diff
@@ -0,0 +1,53 @@
+from fastapi import Body, Depends, FastAPI, HTTPException, status
+from sqlalchemy.orm import Session
+
+from database.crud import validate_fields, write_to_db
+from database.database import Base, engine, get_database
+from database.models import NotificationData
+from database.schemas import Notification
+from utils.values import CreateSubscriptionValues, NotificationValidationValues
+
+Base.metadata.create_all(bind=engine)
+
+app = FastAPI()
+
+
+@app.post(
+    path="/create-notifications-subscription",
+    description=CreateSubscriptionValues.create_subscription_description_message,
+)
+async def create_subscription_to_notifications(
```

Comment:
make endpoint name shorter -> `subscribe_to_notification` is easier and at the same time give you context